yarn.lock

```diff
@@ -4222,7 +4222,7 @@ lodash.uniq@^4.5.0:
   resolved "https://registry.npmjs.org/lodash.uniq/-/lodash.uniq-4.5.0.tgz"
   integrity sha1-0CJTc662Uq3BvILklFM5qEJ1R3M=
 
-lodash@^4.13.0, lodash@^4.16.2, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.21, lodash@^4.17.4, lodash@^4.3.0:
+lodash@^4.13.0, lodash@^4.16.2, lodash@^4.17.10, lodash@^4.17.11, lodash@^4.17.14, lodash@^4.17.15, lodash@^4.17.4, lodash@^4.3.0:
```

Comment:
Oh, does this mean packages we use have dependencies on all these other versions of lodash?

Reply:
Those were all sharing `lodash@4.17.21` (and most of them went away with the `electron-forge` removal).

`yarn why lodash` now outputs
```
=> Found "lodash@4.17.21"
info Reasons this module exists
   - "eslint" depends on it
   - Hoisted from "eslint#lodash"
   - Hoisted from "eslint#table#lodash"
   - Hoisted from "eslint#inquirer#lodash"
   - Hoisted from "electron#@electron#get#global-tunnel-ng#lodash"
   ```